src/components/templates/FireBarrel/FireBarrel.tsx

```diff
@@ -24,10 +23,11 @@ export interface FireBarrelProps {
 export const FireBarrel: React.FC<FireBarrelProps> = ({ venue }) => {
   const { userId, userWithId } = useUser();
 
-  const { room, participants } = useVideoRoomState({
-    user: userWithId,
-    roomName: venue?.name,
-  });
+  const {
+    localParticipant,
+    participants,
+    renderErrorModal,
+  } = useVideoRoomState(userId, venue?.name);
```

Comment:
venue name, not ID?

Reply:
Yep, that's how it worked before refactoring. Changed it to use venue.id everywhere in https://github.com/sparkletown/sparkle/pull/2369/commits/e3fb396a321fe5ad2e70c112614770f643c07016 as its seems more robust.